core/images/mediatypes.go

```diff
@@ -59,6 +59,9 @@ const (
 	MediaTypeImageLayerEncrypted     = ocispec.MediaTypeImageLayer + "+encrypted"
 	MediaTypeImageLayerGzipEncrypted = ocispec.MediaTypeImageLayerGzip + "+encrypted"
 
+	// EROFS media type.
+	MediaTypeErofsLayer = "application/vnd.erofs.layer.v1"
```

Comment:
since it's refined as `application/vnd.erofs.layer.v1`, can we just fix isErofsMediaType() in `plugins/diff/erofs/differ.go` together for easy backporting?